### ISSUE TITLE
NEXT: Implement Zag-based animations for the React Accordion component

### ISSUE THREAD

**Issue by endigo9740** [skeletonlabs/skeleton#2810]
Following our recent [integration of Zag.js](https://github.com/skeletonlabs/skeleton/pull/2778), the React Accordion component animations are no longer working. This is due to Framer Motion being removed as a dependency. We should take steps to replace this with the Zag-based solution.
https://zagjs.com/components/react/accordion

Note this may include may require making use of the Zag Presence feature:
https://zagjs.com/components/react/presence

**Comment by endigo9740** [comment]
I'm going to go ahead and close this out, as we have a definitive answer to this in the upcoming Skeleton v4 release. I'd recommend checking out the updated Fundamentals docs and/or Contributor > Component docs for details when v4 launches for more information.

TL;DR: you'll be able to implement any animation method you wish (Svelte Transitions, native CSS, Motion, etc)
